addon/styles/partials/update-availability/_fixed.scss

```diff
@@ -0,0 +1,103 @@
+.update-availability-fixed {
+  @include fade-in();
+  animation-duration: 1s;
+  padding: 30px 0 20px 0;
+  position: relative;
+
+  .options {
+    &__none {
+      font-size: 20px;
+      margin: 20px 0;
+    }
+
+    &__add {
+      @include flex();
+      @include align-items-center();
+      margin-top: 20px;
+
+      & > div {
+        margin: 0 10px;
+      }
+    }
+  }
+
+  .option {
+    @include flex();
+    @include align-items-center();
+    margin: 20px 0;
+
+    &__label {
+      width: 150px;
+
+      p {
+        font-size: 16px;
+      }
+    }
+
+    &__link {
+      margin-left: 20px;
+    }
+
+    &--medium {
+      .ember-power-select-option {
+        &[data-option-index="0"] {
+          background: color('dropdown', 'item-hover') !important;
+          color: color('medium-gray') !important;
+          cursor: default;
+        }
+      }
+    }
+
+    &--low {
+      .ember-power-select-option {
+        &[data-option-index="1"] {
+          background: color('dropdown', 'item-hover') !important;
+          color: color('medium-gray') !important;
+          cursor: default;
+        }
+      }
+    }
+
+    &--none {
+      .ember-power-select-option {
+        &[data-option-index="2"] {
+          background: color('dropdown', 'item-hover') !important;
+          color: color('medium-gray') !important;
+          cursor: default;
+        }
+      }
+    }
+  }
+
+  .ember-power-select-placeholder {
+    font-size: 12px;
+  }
+
+  .ember-basic-dropdown-trigger {
+    padding: 7px;
+  }
+
+  p {
+    font-size: 14px;
+    font-weight: 500;
+  }
+
+  .align-top-left {
+    top: 25px;
+    left: 10px;
+    font-size: 15px;
+    color: color('text', 'subheading');
+  }
+
+  .uni-input-price {
+    width: 300px;
+  }
+
+  .ember-power-calendar {
+    position: absolute;
+    background-color: white;
```

Comment:
add this to the color pallete of ember-cli-uniq... 'availability',  'background'